models/discriminators.py

```diff
@@ -139,12 +115,10 @@ def loss(self, predictions_on_real, predictions_on_fake):
                 predictions_on_real : Tensor
                 predictions_on_fake : Tensor
         """
-        real_loss = tf.nn.sigmoid_cross_entropy_with_logits(labels=tf.ones_like(predictions_on_real), logits=predictions_on_real)
+        real_loss = tf.nn.sigmoid_cross_entropy_with_logits(labels=tf.fill(predictions_on_real.shape, 0.9), logits=predictions_on_real)
```

Comment:
I don't really understand this change. Why use 0.9 rather than 1?

Reply:
Took it from a repo, I think I will change it back actually it was more just for testing.